js/localgov-publications.js

```diff
@@ -15,3 +15,54 @@
     }
   };
 })(jQuery, Drupal);
+
```

Comment:
All this code should be inside the closure that the existing JS code defines.

Reply:
Actually, I've just realised the existing code is only for admin pages. This new code should probably be in its own file and library definition.